docs/Troubleshooting.BuildingTheApp.md

```diff
@@ -13,7 +13,27 @@ For troubleshooting of other issue, refer to our [troubleshooting index](Trouble
 
 ## Android
 
-### Problem: minSdkVersion mistmatch
+### Problem: AAPT - resource linking failure
+
+For build errors involving AAPT resource linking failure, such as this one:
+
+```text
+Execution failed for task ':app:processReleaseAndroidTestResources'.
+> A failure occurred while executing com.android.build.gradle.internal.res.LinkApplicationAndroidResourcesTask$TaskAction
+   > Android resource linking failed
+     ERROR:: AAPT: error: resource style/Widget.AppCompat.TextView ...
+```
+
+Ensure that the following line appears in your app's `buildscript` (`android/app/build.gradle`) in the `dependencies` section:
+
+```groovy
+dependencies {
+    // ...
+    implementation 'androidx.appcompat:appcompat:1.1.0'
```

Comment:
@jonathanmos Sorry I haven't mentioned this before (although it _did_ come to mind): Using the static `1.1.0` version is tricky here. I think we should add some kind of an indication saying that `1.1.0` may not be the latest/most suitable version for the user's project, etc.